Dockerfile.rhel7

```diff
@@ -13,7 +13,7 @@ RUN --mount=type=cache,target=/go/rhel9/.cache,z \
     make install DESTDIR=./instroot-rhel9 && tar -C instroot-rhel9 -cf instroot-rhel9.tar .
 
 # Add a RHEL 8 builder to compile the RHEL 8 compatible binaries
-FROM registry.ci.openshift.org/ocp/builder:rhel-8-golang-1.22-builder-multi-openshift-4.18 AS rhel8-builder
+FROM registry.ci.openshift.org/ocp/builder:rhel-8-golang-1.23-openshift-4.19 AS rhel8-builder
```

Comment:
While we're updating this, there's one more location below that still references 4.18 builder, could you update that as well and we can close https://github.com/openshift/machine-config-operator/pull/4732/files#diff-c44781af3185b1c38c89ce764336cc7b6ac747b84aab6f8ba2bc5a956181095bR27

@djoshy checking on your comments in https://issues.redhat.com/browse/OCPBUGS-44429, is this builder correct?

Reply:
resolved